stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -232,7 +232,7 @@ match ret9 {
 
 ### Get proxy table
 
-A proxy for a database table that allows performing add/remove operations over the actual database table, can be obtained by `getProxyTable` action as follows.
+Use the `getProxyTable` action shows below to obain a proxy for a database table that allows performing add/remove operations over the actual database table.
```

Comment:
Change "shows below" to "shown below"
Change "obain" to "obtain"

Reply:
Fixed the typos. Thanks!